source/templates/workflows.slim

```diff
@@ -155,7 +145,8 @@
 												.toggleabble-description
 													p[sanitized-markdown="output.shortestDescription()" ng-class="{'oneliner': workflowsCtrl.openedOutputs.indexOf(output) == -1}"]
 													== svg("arrow-down")
-						button.delete-step.rebo.big.hollow-danger[ng-click="workflowsCtrl.deleteStepSelected()" ng-bind="workflowsCtrl.deleteStepButtonText()"]
+						errornotification[message="'#{ data[:strings][:workflows][:steps][:invalid_version][:message] }'" ng-show="!step.isConfigured()"]
```

Comment:
this is a react component wrapped

Reply:
Could this have a custom notation to show that this is a wrapped Bitkit comp? eg. `bk-error-notification`

Reply:
should we use some prefix to indicate "react" instead? I mean bitkit is just a lib which we can use but at the end of the day it's a react component